AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java

```diff
@@ -1606,7 +1607,7 @@ public long fixIntegrity(DeckTask.ProgressCallback progressCallback) {
         ArrayList<String> problems = new ArrayList<>();
         long oldSize = file.length();
         int currentTask = 1;
-        int totalTasks = (mModels.all().size() * 4) + 20; // 4 things are in all-models loops, 20 things are one-offs
+        int totalTasks = (mModels.all().size() * 4) + 21; // 4 things are in all-models loops, 20 things are one-offs
```

Comment:
nit: should remove the counts from the comment as they will skew as they have now
```suggestion
        int totalTasks = (mModels.all().size() * 4) + 21; // a few fixes are in all-models loops, the rest are one-offs
```